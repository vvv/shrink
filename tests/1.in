a
b
b
c
c
c
d
d
d
d
