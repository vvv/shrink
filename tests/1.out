a
b
b
c
.. Last line repeated 3 times
d
.. Last line repeated 4 times
